test/xpu_api/iterators/predef.iterators/move.iterators/move.iter.ops/move.iter.op.incr/post_incr.pass.cpp

```diff
@@ -23,8 +23,14 @@
 #include "support/utils.h"
 #include "support/test_macros.h"
 
+#include <type_traits>
+
 #if TEST_DPCPP_BACKEND_PRESENT
 
+template <typename It>
+constexpr bool __is_forward_iterator =
```

Comment:
I guess the uglification is not required here. I propose to simply name this constant `is_forward_iterator`.

Reply:
Fixed